plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/TypeUtils.java

```diff
@@ -75,7 +75,7 @@ private TypeUtils() {}
     static String getArrayElementPgTypeName(ConnectorSession session, PostgreSqlClient client, Type elementType)
     {
         if (DOUBLE.equals(elementType)) {
-            return "float";
+            return "float8";
```

Comment:
Good catch. Any idea why it didn't cause issues before?

Reply:
I'm not sure, but it was wrong in the first place - I've examined what type is returned when reading `double` and it is `float8`.